CONTRIBUTING.md

```diff
@@ -0,0 +1,75 @@
+# How to Contribute to LoT
+
+We'd love your help!
+
+LoT is [Apache License 2.0](LICENSE) and accepts contributions via
+GitHub pull requests. This document outlines some of the conventions
+on development workflow, commit message formatting, contact points and
+other resources to make it easier to get your contributions accepted.
+
+## Project structure
```

Comment:
I assume project structure would be very dynamic considering this project is at early stage. Probably we can add this section once we have enough contributors and project is more structured.

Reply:
The idea here is that the overall project structure in its base will follow the same pattern and we can keep on updating the structure here in case whenever it goes stale. Isn't it always better to have it then to never have it at all?